### DISCUSSION TITLE
Unable to set cookies

### DISCUSSION THREAD

**Discussion by francoism90** [spatie/laravel-multitenancy#367] [Q&A]
`cookie('name', 'foo')`, but the cookie isn't created. It seems the cookies get destroyed on each request. I'm using a subdomain, maybe this is the issue?

Thanks

**Comment by masterix21** [comment]
Hi @francoism90. Take a look at the cookie's domain.

**Reply by francoism90** [reply]
@masterix21 Thanks for your reply. :)

Using the facade directly seems to work:
```
Cookie::queue('foo', 'bar')
```
Maybe the helper isn't aware of the domain?
